lib/registries/addon/overview/route.ts

```diff
@@ -1,12 +1,64 @@
 import { action } from '@ember-decorators/object';
 import { service } from '@ember-decorators/service';
+import HeadTagsService from 'ember-cli-meta-tags/services/head-tags';
+import { task } from 'ember-concurrency';
+import config from 'ember-get-config';
+import moment from 'moment';
 
+import Contributor from 'ember-osf-web/models/contributor';
+import Institution from 'ember-osf-web/models/institution';
 import Registration from 'ember-osf-web/models/registration';
 import GuidRoute, { GuidRouteModel } from 'ember-osf-web/resolve-guid/guid-route';
 import Analytics from 'ember-osf-web/services/analytics';
+import MetaTags, { HeadTagDef } from 'ember-osf-web/services/meta-tags';
+import Ready from 'ember-osf-web/services/ready';
+import pathJoin from 'ember-osf-web/utils/path-join';
+
+const { assetsPrefix } = config;
 
 export default class Overview extends GuidRoute {
     @service analytics!: Analytics;
+    @service('head-tags') headTagsService!: HeadTagsService;
+    @service metaTags!: MetaTags;
+    @service ready!: Ready;
+
+    headTags?: HeadTagDef[];
+
+    setHeadTags = task(function *(this: Overview, model: any) {
+        const blocker = this.ready.getBlocker();
+        const registration = yield model.taskInstance as Registration;
+        const contributors = yield registration.loadAll('contributors');
+        const institutions = yield registration.loadAll('affiliatedInstitutions');
+        const license = yield registration.license;
+
+        const image = `${assetsPrefix}engines-dist/registries/assets/img/osf-sharing.png`;
+
+        const metaTagsData = {
+            title: registration.title,
+            description: registration.description,
+            publishedDate: moment(registration.dateRegistered).format('YYYY-MM-DD'),
+            modifiedDate: moment(registration.dateModified).format('YYYY-MM-DD'),
+            identifier: registration.id,
+            url: pathJoin(config.OSF.url, registration.id),
+            image,
+            keywords: registration.tags,
+            siteName: 'OSF',
+            license: license && license.name,
+            author: contributors.map((contrib: Contributor) => contrib.users.get('fullName')),
+            institution: institutions.map((ins: Institution) => ins.get('name')),
+        };
+
+        this.set('headTags', this.metaTags.getHeadTags(metaTagsData));
+        this.headTagsService.collectHeadTags();
+        blocker.done();
+
+        // Tell Zotero head meta tags are ready
+        const ev = new Event('ZoteroItemUpdated', {
```

Comment:
It'd be nice if this were done in a central place, since we'd probably want it on any page with meta tags like this. Maybe a method on `meta-tags` service that wraps `headTagsService.collectHeadTags()`?
```ts
updateHeadTags() {
  this.headTags.collectHeadTags();
  
  // https://www.zotero.org/support/dev/exposing_metadata#force_zotero_to_refresh_metadata
  const ev = new Event('ZoteroItemUpdated', ...);
  document.dispatchEvent(ev);
}
```
bonus: since `headTags` is then used on a service instead of a route, you can do `@service headTags` instead of `@service('head-tags') headTagsService`